### ISSUE TITLE
Data Size allocation for flow when converting to LMDB 

### ISSUE THREAD

**Issue by adhara123007** [lmb-freiburg/flownet2#188]
In the code "convert imageset and flow" to convert to lmdb format, the number of channels for the flow is 
2. The data size allocated is 2*2*width*height. As, I understand there is a horizontal and vertical component of the flo. Why do we need 2*2*width*height? It should be 2*width*height.

data_size=  3*width*height              // Image 1
                   +3*width*height              // Image 2
                   +2*2*width*height            // Flow
                   +(width*height-1)/8+1;       // Occlusions

**Comment by nikolausmayer** [comment]
You're right, there are 2×width×height components; but each component is stored as a `short`, i.e. 16 bits = **2** bytes. Compare to the image data which is 8 bit per color channel, and the occlusions which are stored as 1 bit per pixel.

We store flows as `short` integers to conserve space (the raw float flows do not compress well). We avoid too harsh quantization artifacts by scaling with 32.0 before the cast.

**Comment by adhara123007** [comment]
Thanks for the clarification.
